### ISSUE TITLE
Support installer partition styling

### ISSUE THREAD

**Issue by cassidyjames** [elementary/stylesheet#873]
I _think_ this was supported by the old stylesheet. We should make sure it works in the new one, or ship the styles in the installer itself.

![Photo of unstyled partitions in Installer](https://user-images.githubusercontent.com/611168/97752828-66733e80-1aba-11eb-990d-4933eb4bfd48.jpg)

**Comment by danrabbit** [comment]
Yeah it looks like we did it here: https://github.com/elementary/stylesheet/blob/5.4.2/elementary/gtk-3.0/granite-widgets.css#L349

I wonder if we should actually change these styles to use outset gradient and a single background color so that we can just assign background colors in the installer and not carry these specific styles

**Comment by cassidyjames** [comment]
@danrabbit that makes sense to me.

**Comment by danirabbit** [comment]
Transferring to Installer in case there's still something to do here, but these are app-specific styles so we shoudn't have them in the system stylesheet
